source/input.js

```diff
@@ -52,6 +52,10 @@ export default class Input extends Component
 		//  to make sure it works correctly with `redux-form`)
 		onBlur : PropTypes.func,
 
+		// Set 'onKeyDown' handler
+		// Can be used in special cases to handle e.g. enter pressed
+		onKeyDown: PropTypes.func,
```

Comment:
Add a space before the colon, add a dot after `Set 'onKeyDown' handler`. Replace the unary quotes with github code symbol (the one on the tilde keyboard key, the backtick)